srcpkgs/batsignal/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'batsignal'
+pkgname=batsignal
+version=1.1.2
+revision=1
+archs=noarch
```

Comment:
This isn't noarch at all. It is a compiled package. If there aren't architecture constraints, you don't need to define this variable.

Reply:
Hum so archs is only for binaries ?

I have updated the commit , doesn't pass on all tests unfortunately in architecture that aren't x86_64

Reply:
`noarch` is used for a package like `0ad-data`, which only contains data. `archs` can also be used to restrict a package to a certain arch, for example if it only works on x86_64.